<launch>

  <param name="odomTopic" value="/spot/odometry"/>
  <param name="signTopic" value="/signs"/>
  <param name="poseTopic" value="/MCLPose"/>
  <param name="particleTopic" value="/MCLParticles"/>
  <param name="address" value="3 research link, Singapore 119391"/>
  <param name="navGraphPath" value="/ros_ws/data/NUS_navgraph.pickle"/>
  <param name="latlonTopic" value="/gps/fix"/>
  <param name="utmTopic" value="/utm/fix"/>

  <node pkg="signloc_ros" type="NavSatPublisher.py" name="NavSatPublisher" output="screen"/>

  <node pkg="signloc_ros" type="BroadcastNavGraphNode.py" name="BroadcastNavGraphNode" output="screen"/>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find signloc_ros)/rviz/demo.rviz"/> 

  <node pkg="tf2_ros" type="static_transform_publisher" name="static_tf_fake" args="0 0 0 0 0 0 map base_link" />


   <node pkg="signloc_ros" type="MCLNode" name="MCLNode0" output="screen"/> 


</launch>